bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2016B/To2016B.scala

```diff
@@ -193,6 +202,39 @@ object To2016B extends Migration {
       case _         => sys.error("Unknown named target: " + name)
     })
 
+  // Construct a note describing the old conic target
+  def conicTargetNote(ps: ParamSet): Option[(String, String)] =
+    for {
```

Comment:
It seems the note will only be produced if all the params are present. Is that always the case? Could/should support the case when some of the params are missing?